libobs-opengl/gl-egl-common.c

```diff
@@ -46,6 +46,16 @@ typedef unsigned long drm_handle_t;
 
 #endif
 
+#define DRM_FORMAT_RESERVED ((1ULL << 56) - 1)
+
+#define DRM_FORMAT_MOD_VENDOR_NONE 0
+
+#define fourcc_mod_code(vendor, val)                       \
+	((((__u64)DRM_FORMAT_MOD_VENDOR_##vendor) << 56) | \
+	 ((val)&0x00ffffffffffffffULL))
+
+#define DRM_FORMAT_MOD_INVALID fourcc_mod_code(NONE, DRM_FORMAT_RESERVED)
+
```

Comment:
These are defined in `libdrm/drm_fourcc.h`. Is it possible to include that instead of redefining it here?